src/display/canvas.js

```diff
@@ -191,6 +191,10 @@ function mirrorContextOperations(ctx, destCtx) {
 }
 
 function addContextCurrentTransform(ctx) {
+  if (ctx._transformStack) {
+    // Reset the transform stack.
+    ctx._transformStack = [];
+  }
```

Comment:
With these changes, do you still need line 212 below since that one essentially looks like a duplicate now?

Reply:
I don't think so, because then we'd end up attached a _transformStack on canvases in Firefox which don't use this code.

Reply:
In that case, shouldn't this new block of code simply be removed?
Note that the transform stack is already being forcibly reset further down, in non-Firefox browsers, which then ought to render this newly added code unnecessary: https://github.com/mozilla/pdf.js/blob/d9a3a243533de06a61b94765178c3e4d826c8a43/src/display/canvas.js#L208